src/main/java/Calculator.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Calculator {
+    static private final String ADD_OPERATION = "+";
+    static private final String SUBTRACT_OPERATION = "-";
+    static private final String MULTIPLY_OPERATION = "*";
+    static private final String DIVIDE_OPERATION = "/";
+    static private final int NUMBER_INDEX = 0;
+    static private final int OPERATION_INDEX = 1;
+
+    List<String> numberList = new ArrayList<>();
+    List<String> operationList = new ArrayList<>();
+    int result = 0;
+
+    public int operation(int firstNumber, int secondNumber, String operation) {
+        switch (operation) {
+            case ADD_OPERATION:
+                result = firstNumber + secondNumber;
+                break;
+            case SUBTRACT_OPERATION:
+                result = firstNumber - secondNumber;
+                break;
+            case MULTIPLY_OPERATION:
+                result = firstNumber * secondNumber;
+                break;
+            case DIVIDE_OPERATION:
+                result = firstNumber / secondNumber;
+                break;
+            default:
+                throw new RuntimeException("부적합한 연산자가 입력되었습니다.");
+        }
+        return result;
+    }
+
+    public int execute(String[] line) {
+        validation(line);
+        addNumberList(line);
+        addOperationList(line);
+        result = Integer.parseInt(numberList.get(0));
+        for (int i = 0; i < operationList.size(); i++) {
+            result = operation(result, Integer.parseInt(numberList.get(i + 1)), operationList.get(i));
+        }
+        return result;
+    }
+
+    public void validation(String line[]) {
```

Comment:
```suggestion
    public void validation(String[] line) {
```
Array의 선언은 일반적으로 타입뒤에 대괄호를 붙입니다. 아래도 동일하게 변경해주세요.